### DISCUSSION TITLE
Next question and I sincerely hope this [May 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5714] [Discord Threads]
> 🙋‍♂️ **phrawzty**

**Comment by o1lab** [comment]
> 🙋‍♂️ **phrawzty**  
#### Node: **v16.19.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.106.0**  
#  
> 🙋‍♂️ **wkw**  
#### have u applied any filters on the grid before sharing?  
#  
> 🙋‍♂️ **phrawzty**  
#### <@495466997304262658> That was my first thought also. In this case: no. No filters. Not even hiding any fields.  
#  
> 🙋‍♂️ **wkw**  
#### i shared a grid view and im able to see the data. any error logs u can share?  
#  
> 🙋‍♂️ **phrawzty**  
#### <@495466997304262658> What is the canonical way to extract logs? Javascript console? Syslog? The documentation doesn't make it clear. 😦  
![image0](https://cdn.discordapp.com/attachments/1096726919099203696/1096874951308480703/image.png)  
#  
> 🙋‍♂️ **wkw**  
#### normally u can check browser console log n the server log  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1096726919099203696)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
